ExerciseMVC/ExerciseMVC/Controllers/HomeController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using ExerciseMVC.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace ExerciseMVC.Controllers
+{
+    public class HomeController : Controller
+    {
+        private TaskDBContext db = new TaskDBContext();
+
+        public ActionResult Index()
+        {
+            return View(db.Tasks.ToList());
+        }
+
+        [HttpPost]
+        public ActionResult Index(FormCollection values)
+        {
+            var newTask = new Task() { TaskDescription = values["TaskDescription"], TaskDate = DateTime.Now, TaskCategory = (Category)Enum.Parse(typeof(Category), values["TaskCategory"]) };
```

Comment:
Doesn't matter either way but you could split this into multiple lines for readability
```suggestion
            var newTask = new Task() 
            {
              TaskDescription = values["TaskDescription"], 
              TaskDate = DateTime.Now, 
              TaskCategory = (Category)Enum.Parse(typeof(Category), 
              values["TaskCategory"]) 
            };
```